docs/content/about-iota/iota-wallet/how-to/integrate-ledger.mdx

```diff
@@ -26,6 +29,21 @@ To use IOTA Wallet with Ledger, install the IOTA Rebased app on your device thro
 5. Click **Install** to download the IOTA Rebased app to your device.
 6. Your device will show the installation progress.
 
+### Manually (for deprecated devices)
```

Comment:
Right now this is only ledger nano s, why not mention its name instead of `deprecated devices`?

Reply:
I just wanted to be future proof but I can change

Reply:
I think it would help people to find it, maybe just add it `for deprecated devices such as Ledger Nano S`

Reply:
done, thanks